internal/metrics/prometheus.go

```diff
@@ -152,6 +156,20 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey},
 	)
 
+	controllerMetrics.TranslationDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name: MetricNameTranslationDuration,
+			Help: fmt.Sprintf(
+				"Duration of  translating Kubernetes resources intoo Kong state. "+
+					"`%s` describes whether there were unrecoverable errors (`%s`) or not (`%s`). "+
+					"Unrecoverable error in this case means KIC wasn't able to translate a Kubernetes object to Kong model.",
+				SuccessKey, SuccessFalse, SuccessTrue,
+			),
+			Buckets: prometheus.ExponentialBucketsRange(1, float64(time.Hour.Milliseconds()), 30),
```

Comment:
IMHO hour is a bit to much as timeouts would kick in anyway. WDYT about using 30s or a minute here?

Reply:
Updated to 1 min.